src/main/java/cubicchunks/lighting/SkyLightCubeDiffuseProcessor.java

```diff
@@ -57,7 +57,7 @@ private void process(Cube cube, long address) {
 
 		Cube.LightUpdateData data = cube.getLightUpdateData();
 
-		Column column = cache.getColumn(columnX, columnZ);
+		Column column = cache.provideChunk(columnX, columnZ);
```

Comment:
FirstLightProcessor should be called only when all needed cubes already exist, if Column doesn't exist it will crash anyway